enterprise-suite/scripts/lbc_test.py

```diff
@@ -131,15 +131,13 @@ def test_install(self):
         expect_cmd(r'helm repo add es-repo https://repo.lightbend.com/helm-charts')
         expect_cmd(r'helm repo update')
         expect_cmd(r'helm status enterprise-suite', returncode=-1)
-        expect_cmd(r'kubectl get pvc --namespace=lightbend --no-headers')
         expect_cmd(r'helm install es-repo/enterprise-suite --name enterprise-suite --namespace lightbend --devel --values \S+')
-        lbc.main(['install', '--skip-checks', '--creds='+self.creds_file])
+        lbc.main(['install', '--skip-checks', '--nowarn', '--creds='+self.creds_file])
```

Comment:
This all needs refinement. I just sprinkled in `--nowarn` to allow things to work as is.

Reply:
Maybe a simple improvement could be to extract all the repeating params into a list, since there's a few now:
```
lbc.main(install_common +  ['--export-yaml=console'])
```

Reply:
Good idea.  I'll look at that when I figure out how to add more tests for these changes (in a separate PR).  (A bit of a mystery to me atm.)